timeline/urls.py

```diff
@@ -1,8 +1,13 @@
 from django.urls import path
 from .views import (
-    TimelineCreateView
+    TimelineCreateView,
+    TimelineDeleteView,
+    TimelineDetailView
 )
 
 urlpatterns = [
     path('', TimelineCreateView.as_view(), name='timeline-create'),
+    path('<uuid:id>/', TimelineDetailView.as_view(), name='timeline-detail'),
+    path('<uuid:id>/delete/', TimelineDeleteView.as_view(), name='timeline-delete'),
+    path('create/', TimelineCreateView.as_view(), name='timeline-create'),
```

Comment:
I agree with Catherine, this isn't necessary at all because you already used APIView then just use request method to differentiate between it's feature.